images/tensorflow/Dockerfile

```diff
@@ -12,23 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG image_version=nightly
+ARG image_version="nightly"
+
 FROM tensorflow/tensorflow:${image_version}-gpu
```

Comment:
Current RedVM has python:3.9.16-bullseye. Therefore this need to be

FROM python:3.9.16-bullseye